src/Components/SmartComponents/SystemsPage/SystemsPage.js

```diff
@@ -40,7 +53,11 @@ const SystemsPage = () => {
     const selectedRowsRawData = useSelector(({ entities }) => entities?.selectedRowsRawData || []);
     const isLoaded = useSelector(({ entities }) => entities?.loaded || false);
 
-    const parameters = useSelector(({ SystemsPageStore }) => SystemsPageStore.params, shallowEqual);
+    let parameters = useSelector(({ SystemsPageStore }) => SystemsPageStore.params, shallowEqual);
+    if (!canReadExcludedIncluded) {
+        parameters.excluded = '';
```

Comment:
When user does not have `canReadExcludedIncluded` permission, he should be able to see only included systems (hide all excluded systems) so this filter should be set to false
`parameters.excluded = 'false';`

This is actually current default filter so you don't have to change the parameter, just don't let user change it and don't show filter chip (act like there's no filter applied at all)

Reply:
I thought that excluded systems will be not loaded anyway, so I hid the filter with changing the default value for excluded, but yea I can apply this logic in filter chip so excluded filter is applied, but not seen in case of `canReadExcludedIncluded` being false.

But maybe we should think, if this is a good solution, because if user changes something in code on FE then he will be able to see excluded systems anyway right?

Reply:
This is a good observation, but actually, no he won't be, backend will be checking permissions too before returning restricted data. Currently this check is disabled on the backend by a feature flag, until whole granular RBAC is implemented on the front-end. So after this feature flag is enabled, user won't be able to access restricted data, even if he tampered with FE code.